.github/workflows/continuous-integration.yml

```diff
@@ -89,35 +64,24 @@ jobs:
           environment-file: environment-noplots.yml
           python-version: 3.11
           auto-activate-base: false
+          conda-remove-defaults: true
```

Comment:
We already test conda in another workflow. Can we go back to `apt-get`?

Reply:
The `gfortran` compiler used in these builds comes from the system (no need to `apt-get` it). The Python environment only supports the Python scripts used in testing.

Reply:
Good catch, thanks, `ninja` and `libnetcdff-dev` now come from `apt-get`.